lib/archethic/bootstrap/network_constraints.ex

```diff
@@ -0,0 +1,134 @@
+defmodule Archethic.Bootstrap.NetworkConstraints do
+  @moduledoc """
+  This GenServer make sure that Node load network constraints before any transaction validation.
+  Updates gen_addr constraints post bootstrap, if fails wait for node to get authorized
+  """
+
+  use GenServer, restart: :transient
```

Comment:
I think you don't need to use a GenServer anymore as you don't get the NSS seed anymore.
Now you fetch the NSS and reward chain from DB and get the genesis address, so you can diretly persist addresses at the end of bootstrap no ?
I tried with the same function than in NetworkConstraints, it works well

Reply:
How can we get nil from DB as all these chain are initialized during network init, or downloaded during bootstrap self repair ?

Reply:
Yes I had the same question, so we  loop until we persist.(on node udpate)
If we dont loop it will crash , may also crash bootstrap process , and restrict node in validation of tx.
So need of independent transient process.